generators/chipyard/src/main/scala/harness/HarnessBinders.scala

```diff
@@ -106,7 +106,7 @@ class WithBlockDeviceModel extends OverrideHarnessBinder({
 class WithLoopbackNIC extends OverrideHarnessBinder({
   (system: CanHavePeripheryIceNIC, th: HasHarnessInstantiators, ports: Seq[ClockedIO[NICIOvonly]]) => {
     implicit val p: Parameters = GetSystemParameters(system)
-    ports.map { n => NicLoopback.connect(Some(n.bits), p(NICKey)) }
+    ports.map { n => NicLoopback.connect(Some(n.bits), p(NICKey), n.clock, th.harnessBinderReset.asBool) }
```

Comment:
If you just do `withClockAndReset(...) { NiCLoopback.connect(...) }` its a bit cleaner